mscr-ui/src/modules/form/validate-form.tsx

```diff
@@ -7,6 +7,7 @@ import {
   formatsAvailableForSchemaRegistration,
 } from '@app/common/interfaces/format.interface';
 
+
```

Comment:
Is there a reason for adding extra empty lines?

Reply:
Nope, just accidental !!